packages/core/src/App/Components/Elements/login-prompt.jsx

```diff
@@ -36,10 +35,5 @@ LoginPrompt.propTypes = {
     page_title: PropTypes.string,
 };
 
-// TODO - Remove this default setting once sign-up has been integrated to app 2
-LoginPrompt.defaultProps = {
-    onSignup: () => { window.open(urlFor('new-account', undefined, undefined, true)); },
-};
-
```

Comment:
Did you mean to make this change in the Trader package?

Reply:
The path is placed up to this comment:
packages/core/src/App/Components/Elements/login-prompt.jsx
it's on core package.

Reply:
your change should be made in trader, not core :)

Reply:
OK since they were the exact same, I thought we should do this on both. NP I will make it correct :)

Reply:
could you revert it to how it used to be? now the line orders have been changes